src/screens/Page3/CountDown/index.js

```diff
@@ -0,0 +1,35 @@
+import React, { useEffect, useState, useRef } from 'react';
+import { View, Text } from 'react-native';
+import { Button } from 'react-native-elements';
+
+import styles from './styles';
+
+const Comp = ({ duration, onComplete, onCancel }) => {
+  const [count, setCount] = useState(duration);
+  const timerRef = useRef();
+
+  useEffect(() => {
+    timerRef.current = setInterval(() => {
+      setCount((v) => v - 1);
+    }, 1000);
+  }, []);
+
+  if (count <= 0) {
```

Comment:
Use `useEffect` to trigger `onComplete` so that it will be triggered only once instead of triggered on every render.